src/components/MajorFilter.jsx

```diff
@@ -6,7 +6,7 @@ import PropTypes from 'prop-types';
 import ArrowDropUpIcon from '@mui/icons-material/ArrowDropUp';
 import SearchIcon from '@mui/icons-material/Search';
 
-function MajorFilter({ majors, onChange }) {
+function MajorFilter({ majors, onChange, majorParams }) {
```

Comment:
used in order to distinguish which filteres are currently applied and which ones are still pending.